addons/binding/org.openhab.binding.loxone/src/main/java/org/openhab/binding/loxone/internal/core/LxControlDimmer.java

```diff
@@ -152,8 +152,7 @@ private Double mapOHToLoxone(Double ohValue) {
             Double max = getMax();
             Double min = getMin();
             if (max != null && min != null) {
-                double value = min + (ohValue / ((max - min) / 100));
```

Comment:
This formula is wrong today - it should read:
double value = min + ohValue * (max - min) / 100;